.gitignore

```diff
@@ -3,6 +3,10 @@
 *
 !*/
 
+# Ignore build directories generated by default MSVC CMake integration
+# (otherwise causes terribly slow indexing)
+out/
+
```

Comment:
because when I open ftxui using MSVC's Open Folder option, it does mkdir out in the project's folder, and stores all built files there by default. From what I can gather, you ignore all files (*), but not all folders (!*/). To ignore all folders, you have to add */

Reply:
Okay. This sounds good to me then.